.github/workflows/nightly-ui-tests-checks.yml

```diff
@@ -0,0 +1,27 @@
+name: "UI Tests"
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
These shouldn't be here, double check the `pull` and `pull_request` options are removed before merging